index.html

```diff
@@ -1,45 +1,45 @@
-					</section>
-					<section id="info">	
-						<h2>About Me</h2>
-						<p class="interests">I was born in New York City and have spent much of my career up until now following my passion in wildlife conservation and eco-tourism. My favorite animal is the clouded leopard, pictured below, and I hope to one day travel to Southeast Asia and try to see one in the wild. In addition to my love for animals and ecology I am also a rabid team sports fan, supporting most New York based clubs in addition to Borussia Dortmund for European football.</p>
-					</section>
-					<section class="photos">
-						<img class="leopard" src="20190909_0624sp.jpg">
-						<img class="bball" src="1244979430.0.jpg">
-					</section>
-				</div>
-				<div id="portfolio">
-						<h2>Portfolio</h2>
-						<p class="projects">These are a few of the projects I have completed during the program.</p>
-					<section class="box-one">
-						<p class="project-description1">Project 1</p>
-						<a class="git-link" href="https://github.com" target="_blank">GitHub</a>
-					</section>
-					<section class="box-two">
-						<p class="project-description2">Project 2</p>
-						<a class="git-link" href="https://github.com" target="_blank">GitHub</a>
-					</section>
-				</div>
+		<div class="about-me">
+			<h1>Welcome to My Page!</h1>
+			<section class="me">	
+				<img class="personal-photo" src="IMG_0368.jpg">
+				<p class="intro">Greetings, my name is Colin Meier and this webpage will be used to showcase the work I have completed throughout my time at iO Academy.</p>
+			</section>
+			<section id="info">	
+				<h2>About Me</h2>
+				<p class="interest-description">I was born in New York City and have spent much of my career up until now following my passion in wildlife conservation and eco-tourism. My favorite animal is the clouded leopard, pictured below, and I hope to one day travel to Southeast Asia and try to see one in the wild. In addition to my love for animals and ecology I am also a rabid team sports fan, supporting most New York based clubs in addition to Borussia Dortmund for European football.</p>
+			</section>
+			<section class="about-me-photos">
+				<img class="interests" alt="Leopard" src="20190909_0624sp.jpg">
+				<img class="interests" alt="Basketball" src="1244979430.0.jpg">
+			</section>
+		</div>
+		<div id="portfolio">
+			<h2>Portfolio</h2>
+				<p class="projects">These are a few of the projects I have completed during the program.</p>
+			<section class="box">
+				<p class="project-description">This is a description of my first project.</p>
+				<a class="git-link" href="https://github.com" target="_blank">GitHub</a>
+			</section>
+			<section class="box">
+				<p class="project-description">This is a description of my second project.</p>
+				<a class="git-link" href="https://github.com" target="_blank">GitHub</a>
+			</section>
```

Comment:
box is better than box-one and box-two, but still as a name it is too general to be semantic, because many things on the page can be described as a box. needs a class name that describes the element's purpose